schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonOpenDataSoft.xsl

```diff
@@ -562,10 +562,9 @@
           <cit:linkage>
             <gco:CharacterString>
               <xsl:value-of select="concat(nodeUrl,
-                                   '/explore/dataset/',
+                                   '/api/explore/v2.1/catalog/datasets/',
                                    (datasetid|dataset/dataset_id)[1],
-                                   '/download?format=', $format,
-                                   '&amp;timezone=Europe/Berlin&amp;use_labels_for_header=false')" />
+                                   '/exports/', $format)" />
```

Comment:
I thin adding the param `use_labels=true` would be beneficial as the resulting file would have human readable column names.

Reply:
Thanks @jahow , I amended the param in the previous commit.